packages/itwin/tree-widget/src/test/trees/models-tree/ModelsTree.test.tsx

```diff
@@ -258,23 +254,21 @@ describe("ModelsTree", () => {
       it("should invoke visibilityHandler factory when it is passed as a function", async () => {
         const node: TreeNodeItem = { id: "test", label: PropertyRecord.fromString("test-node"), isCheckboxVisible: true };
         setupDataProvider([node]);
-        const visibilityHandlerCallback = () => {return visibilityHandlerMock;};
-        const visibilityHandlerSpy = sinon.spy(visibilityHandlerCallback);
-        const result = render(<ModelsTree {...sizeProps} iModel={imodelMock.object} modelsVisibilityHandler={visibilityHandlerCallback} activeView={mockViewport().object} />);
+        const visibilityHandlerSpy = sinon.stub().returns(visibilityHandlerMock);
+        const result = render(<ModelsTree {...sizeProps} iModel={imodelMock.object} modelsVisibilityHandler={visibilityHandlerSpy} activeView={mockViewport().object} />);
         await waitFor(() => result.getByText("test-node"), { container: result.container });
         expect(visibilityHandlerSpy.calledOnce);
```

Comment:
I believe this doesn't check anything

```suggestion
        expect(visibilityHandlerSpy).to.be.calledOnce;
```